STUDIO/org.openl.rules.jackson/src/org/openl/rules/serialization/spr/SpreadsheetSerializer.java

```diff
@@ -0,0 +1,52 @@
+package org.openl.rules.serialization.spr;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+import org.openl.rules.calc.SpreadsheetResult;
+
+/**
+ * Default SpreadsheetResult serializer for logging purposes.
+ *
+ * @author Yury Molchan
+ */
+public class SpreadsheetSerializer extends StdSerializer<SpreadsheetResult> {
```

Comment:
Shouldn't we use ``CustomSpreadsheetResult`` representation instead of the original one? Original SpreadsheetResult is hard to read for humans

Reply:
`ProjectJacksonObjectMapperFactoryBean` is used now.